docs/guide/tutorials/projections.txt

```diff
@@ -1,3 +1,88 @@
 ===========
 Projections
 ===========
+
+By default, queries in MongoDB return all fields in matching
+documents. To limit the amount of data that MongoDB sends to
+applications, you can include a projection document in the query
+operation.
+
+Projection Document
+-------------------
+
+The projection document limits the fields to return for all
+matching documents. The projection document can specify the
+inclusion of fields or the exclusion of fields and has the
+following form:
+
+.. code-block:: js
+
+   { field1: <value>, field2: <value> ... }
+
+``<value>`` may be ``0`` (or ``false``\ ) to exclude the field, or
+``1`` (or ``true``\ ) to include it. With the exception of the ``_id``
+field, you may not have both inclusions and exclusions in the same
+projection document.
+
+Examples
+--------
+
+The following code example uses the ``restaurants`` sample dataset.
+
+To return only the ``name``\ , ``cuisine`` and ``_id\ ``fields for documents
+which match the query filter, explicitly include the``\ name\ ``and``\ cuisine\ ``fields in the projection document. The``\ _id`` field is
+included automatically unless specifically excluded.
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   
+   // Connection URL
+   const url = 'mongodb://localhost:27017';
+   
+   // Database Name
+   const dbName = 'myproject';
+   
+   // Create a new MongoClient
+   const client = new MongoClient(url);
+
+   // Use connect method to connect to the Server
+   client.connect().then(async function() {
+     console.log("Connected successfully to server");
+   
+     const collection = client.db(dbName).collection('restaurants');
+
+     try {
+       const results = await collection
```

Comment:
variable `results` isn't used. Did you mean to print this out? If not can you remove the variable and just `await collection...`